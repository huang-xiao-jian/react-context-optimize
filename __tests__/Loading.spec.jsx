// external
import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

// internal
import Loading from '../src/components/Loading';

describe('Loading Component', () => {
  beforeAll(() => {
    configure({ adapter: new Adapter() });
  });

  it('should render standard loading', () => {
    const wrapper = shallow(
      <Loading message="waiting...">Jest Unit Test</Loading>
    );

    expect(wrapper.html()).toMatchSnapshot();
  });
});
