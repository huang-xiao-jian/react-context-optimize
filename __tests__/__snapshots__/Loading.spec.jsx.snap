// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component should render standard loading 1`] = `"<article class=\\"polaris\\"><div class=\\"polaris-spinner\\"><div class=\\"polaris-cube polaris-cube--blue\\"></div><div class=\\"polaris-cube polaris-cube--green\\"></div><div class=\\"polaris-cube polaris-cube--pink\\"></div><div class=\\"polaris-cube polaris-cube--yellow\\"></div><div class=\\"polaris-message\\">waiting...</div></div></article>"`;
